views/error404.html

```diff
@@ -0,0 +1,24 @@
+% include('base.html', title="PyJudge")
+<body>
+  <script>
+        var link = document.createElement('link');
+        link.rel = 'stylesheet';
+        link.type = 'text/css';
+        link.href = '/static/css/notfound.css';
+        document.getElementsByTagName('HEAD')[0].appendChild(link);
```

Comment:
oh no no.
it's a good hack but there is a proper way of doing this. Keep at it :+1:

Reply:
i will try to find the proper way also ultil that keep this

Reply:
let's keep the request open until this is fixed